sanic_openapi/doc.py

```diff
@@ -120,7 +120,8 @@ def serialize(self):
             items = serialize_schema(self.items[0])
         return {
             "type": "array",
-            "items": items
+            "items": items,
+            **super().serialize()
```

Comment:
Wouldn't you want to call `super().serialize` first, and then `update()` it with `type` and `items`? That way those keys do not accidentally get overwritten?

Reply:
@ahopkins Makes sense. I just aligned with all other super calls for serialize method in different fields and kept is same. Let me know if you still want the super call as first statement or we can leave it for future when changes would be done for all fields?